### PR TITLE
Add 'script' Go library

### PR DESCRIPTION
**Pull request by bitfield**
Please complete the following checklist if your PR is adding new link to the list:

- [X] I've read [contributing guidelines](https://github.com/MunGell/awesome-for-beginners/blob/master/CONTRIBUTING.md)
- [X] This PR does not introduce duplicates to the list
- [X] The link is added at the bottom of the list category
- [X Suggested repository is maintained, have active community, is able to mentor new contributors and have issues with the suggested label
- [X] The link I add follows the suggested pattern:

```
- [Repository Name](link-to-repository-label) _(label: beginner-friendly-label-in-the-repository)_ <br> Description
```

Example link formatting:

```
- [awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners/labels/good-first-contribution) _(label: good-first-contribution)_ <br> A list of awesome beginners-friendly projects.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Moby](https://github.com/moby/moby/labels/exp%2Fbeginner) _(label: exp/beginner)_ <br> Open-source application container engine
 - [Terraform](https://github.com/hashicorp/terraform/labels/good%20first%20issue) _(label: good first issue)_ <br> A tool for building, changing, and versioning infrastructure safely and efficiently.
 - [TiDB](https://github.com/pingcap/tidb/labels/for%20new%20contributors) _(label: for new contributors)_ <br> A distributed scalable Hybrid Transactional and Analytical Processing (HTAP) database
+- [script](https://github.com/bitfield/script/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) _(label: good first issue)_ <br> A Go library for doing the kind of tasks that shell scripts are good at: reading files, executing subprocesses, counting lines, matching strings, and so on. Beginners are very welcome and will get detailed code review and help through the PR process.
```
Comment (MunGell) [inline README.md]: Could you please change link to https://github.com/bitfield/script/labels/good%20first%20issue

**Comment by bitfield** [comment]
Thanks @MunGell! No sooner said than done!

**Comment by bitfield** [comment]
Hi @MunGell! Is there anything else you'd like me to do before this PR is ready to be merged?

**Comment by MunGell** [comment]
👍

**Pull request merged**